frontend/src/Pages/AdminPage.js

```diff
@@ -44,18 +64,23 @@ export default function AdminPage() {
     return (
       <Page>
         <Loading />
+        <BottomNav />
       </Page>
     )
   }
 
   return (
-    <MainPage>
+    <PageHeader>
+      <Typography variant="h3">{'Hallo ' + user.userName}</Typography>
       <MainAdmin>
-        <Typography variant="h3">{'Hallo ' + user.userName}</Typography>
         <StatsBoxGlobal right={right} wrong={wrong} all={overall} />
+        <StatsBox
+          text={'Wie viele Mitarbeiter nutzen die App?'}
+          percent={Math.round((numberOfUsers * 100) / allUsers)}
```

Comment:
I have seen this snippet twice, maybe extract to a function for reuse?

Reply:
Right, done 👍